src/phoenix/server/bearer_auth.py

```diff
@@ -108,3 +127,35 @@ async def is_authenticated(request: Request) -> None:
         raise HTTPException(status_code=HTTP_401_UNAUTHORIZED, detail="Expired token")
     if claims.status is not ClaimSetStatus.VALID:
         raise HTTPException(status_code=HTTP_401_UNAUTHORIZED, detail="Invalid token")
+
+
+async def create_access_and_refresh_tokens(
```

Comment:
add a similar one for the oauth counterparts?

Reply:
Not sure what you have in mind here. This method is used inside the OAuth2 routers. Not sure we will re-use any of the code related to the OAuth2 authorization code flow elsewhere.

Reply:
ok. it was just a thought